openssl-dynamic/src/main/java/org/apache/tomcat/jni/SSLContext.java

```diff
@@ -312,7 +312,21 @@ public static native boolean setCertificateBio(
     public static native long sessionTimeouts(long ctx);
 
     /**
-     * Set TLS session keys.
+     * TLS session ticket key resumption statistics.
+     */
+    public static native long sessionTicketKeyNew(long ctx);
+    public static native long sessionTicketKeyResume(long ctx);
+    public static native long sessionTicketKeyRenew(long ctx);
+    public static native long sessionTicketKeyFail(long ctx);
```

Comment:
`long` and `int` are ill defined in terms of underlying size in the c specifications. If the underlying data is `uint32` you should use the same type.